src/test/scala/com/gu/support/workers/AccessScopeCodecTest.scala

```diff
@@ -1,19 +1,19 @@
 package com.gu.support.workers
 
-import com.gu.support.workers.model.AccessScope
-import com.gu.support.workers.model.AccessScope._
+import com.gu.support.workers.model.AccountAccessScope
+import com.gu.support.workers.model.AccountAccessScope._
 import org.scalatest.{FlatSpec, Matchers}
 
-class AccessScopeCodecTest extends FlatSpec with Matchers {
+class AccountAccessScopeCodecTest extends FlatSpec with Matchers {
 
   it should "for backwards compatibility, deserialise when it's not specified to no restriction" in {
```

Comment:
Change the name from '... no restriction' to '... authenticated'

Reply:
oops only just saw these, good spot!
I feel like test descriptions are like comments - they are naturally not kept up to date, they are not telling us anything that we shouldn't be able to get from the code, and they can easily get attached to the wrong thing

Reply:
https://github.com/guardian/support-workers/pull/150